<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>set_critical_variable</class>
 <widget class="QWidget" name="set_critical_variable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_set_critical_variable">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Critical Variable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_firing_point">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_summation_temp">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rhs_upper">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rhs_lower">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_attribute_per_core">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_data_division">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>150</y>
     <width>134</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Firing Point change value</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>190</y>
     <width>145</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>summation temp thershold</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>230</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rhs upper</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>data division</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>attributes per core</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>270</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rhs lower</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>110</y>
     <width>129</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Extreme weight remover</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_extreame_weight">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>100</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>30</y>
     <width>145</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Flatening function enabled</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_flatening">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>70</y>
     <width>155</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zero weight remover enabled</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_zero_weight">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
